README.md

```diff
@@ -154,6 +154,13 @@ Should it be desired to fetch an update ouside of these hours, then the API limi
 
 For example, to update just after midnight, as well as take advantage of auto-update, create the desired automation to force update, then reduce the API limit configured in the automation accordingly. (For this exmple, if the API key has ten total calls allowed per day and two rooftop sites, reduce the API limit to eight because two updates will be used when the automation runs.)
 
+> [!NOTE]
+> _Transitioning to auto-update from using an automation:_
+>
+> If currently using the recommended automation, which spreads updates fairly evenly between sunrise and sunset, turning on auto-update from sunrise to sunset should not cause unexpeced forecast fetch failures due to API limit exhaustion. The recommended automation is not identical to auto-update, but is fairly close in timing.
+>
+> If implementing a reduced API limit, plus a futher forced update at a different time of day (like midnight), then a 24-hour period of adjustment may be needed, which could possibly see API use exhaustion (even if the acutal API usage has not actually been exhausted). These errors will clear within 24 hours.
```

Comment:
nitpick / question / suggestion (and, because the comment box shows squiggly lines, there's an **actual** typo, which I'll separately fix): Do you want to say "see API use exhaustion _reported_ (even if the acutal API usage has not actually been exhausted)

acutal

Reply:
Note, since I was there, I made the change and added reported - but you can take it out.  That's what a democracy looks like in a hung parliament :)

Reply:
> Do you want to say ... reported

Yes. Yes I do.